.github/workflows/deploy-hubs.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
           # add its name to the list
           - 2i2c
           - cloudbank
-          # - carbonplan  # We need to edit deployer to connect to aws before we can autodeploy carbonplan
+          - carbonplan
```

Comment:
Do we also need to add an AWS login step into CD? Such as https://github.com/marketplace/actions/configure-aws-credentials-action-for-github-actions

Reply:
From my understanding, we would not need that step... this PR is retrieving the credentials from the encrypted file and setting up the needed AWS environment variables by itself, so any AWS CLI configuration should not be needed.